### PR TITLE
Review tweaks for surveys jQuery removal

### PR DESCRIPTION
**Pull request by andysellick**
## What / why
Changes based on a review that was completed post-merge of https://github.com/alphagov/static/pull/2697 from @kevindew.

Comments marked with a 👍  have been resolved as suggested. Other comments inline in the original review.

## Visual changes
None.

### CODE REVIEW

**Review by kevindew — APPROVED** [review]
Awesome, thanks

**Comment by andysellick** [comment]
@kevindew have updated this with removal of the `document.ready` equivalent as suggested.

File: app/assets/javascripts/surveys.js [hunk 1]
```diff
@@ -496,7 +494,7 @@
       if (breadcrumbs === undefined) {
         return false
       } else {
-        var breadcrumbMatchingExpr = new RegExp($.makeArray(breadcrumbs).join('|'), 'i')
+        var breadcrumbMatchingExpr = new RegExp(breadcrumbs.join('|'), 'i')
```
Comment (kevindew) [inline app/assets/javascripts/surveys.js:497]: Nice catch!

**Review by kevindew — APPROVED** [review]
Nice one

**Pull request merged**